cpp/src/arrow/CMakeLists.txt

```diff
@@ -144,6 +144,10 @@ set(ARROW_SRCS
     result.cc
     scalar.cc
     sparse_tensor.cc
+    sparse_tensor/coo_converter.cc
+    sparse_tensor/csc_converter.cc
+    sparse_tensor/csf_converter.cc
+    sparse_tensor/csr_converter.cc
```

Comment:
Could call this directory "tensor" to make more general?

Reply:
OK, I will do.